### PR TITLE
Issue 3427: fix failing searchs containing :

### PR DESCRIPTION
**Pull request by superalex**

### CODE REVIEW

File: taiga/base/utils/db.py [hunk 1]
```diff
@@ -127,7 +128,102 @@ def update_in_bulk_with_ids(ids, list_of_new_values, model):
...
+        if last == ")":
+            break
+        if last == "(":
+            paren_count -= 1
+        res.pop()
+    while paren_count > 0:
+        res.append(")")
+        paren_count -= 1
+
+    print(" ".join(res))
```
Comment (bameda) [inline taiga/base/utils/db.py]: print(:cat2: :dog2: :bird: :carousel_horse:)

**Comment by bameda** [comment]
Without the print sentence is good for me.

**Comment by superalex** [comment]
Sorry!, I forgot to remove it, it's updated right now ;)

**Comment by bameda** [comment]
Great. merged and backported to stable

**Pull request merged**